openvdb/tools/ParticlesToLevelSet.h

```diff
@@ -758,11 +764,11 @@ struct ParticlesToLevelSet<SdfGridT, AttributeT, InterrupterT>::Raster
     getAtt(size_t n, AttT& a) const { mParticles.getAtt(n, a); }
 
     template <typename T>
-    typename boost::enable_if<boost::is_same<T,ValueT>, ValueT>::type
+    typename boost::enable_if<std::is_same<T,ValueT>, ValueT>::type
     Merge(T s, const AttT&) const { return s; }
 
     template <typename T>
-    typename boost::disable_if<boost::is_same<T,ValueT>, ValueT>::type
+    typename boost::disable_if<std::is_same<T,ValueT>, ValueT>::type
```

Comment:
Should this be `std::enable_if_v<!std::is_same<T, ValueT>::value, ValueT>`? (`std::enable_if_v` is C++14; `std::is_same_v` is C++17.)

In C++17, it looks like these two `Merge` functions could be combined and an `if constexpr` used to switch the behavior.

Reply:
Looks like it would work. I share your wish of being able to use c++17 _v. It could probably be easily added to the std::namespace. 

Anyway, since this PR was made, I'm not exactly sure I'm allowed to contribute as I'd need to get it approved by the company I am contracting for, so feel free to make any changed yourself :)